W-15828285-update-rtf-pod-kt (#1360)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/install-self-managed-network-configuration.adoc

====

Before installing or using Runtime Fabric, ensure that the following ports and hostnames are configured correctly.

> Note that all internal communication within the cluster (between pods and services) needs to be allowed.

Reviewer: ```suggestion
[NOTE]
You must allow internal communications within the cluster between pods and services.
```
